support-frontend/test/selenium/subscriptions/CheckoutsSpec.scala

```diff
@@ -82,7 +89,7 @@ class CheckoutsSpec extends AnyFeatureSpec
     assert(checkoutPage.pageHasLoaded)
 
     Given("The user fills in their details correctly")
-    checkoutPage.fillForm
+    checkoutPage.fillForm()
```

Comment:
These added parentheses are just to keep Intellij's code analysis happy

Reply:
👍